src/App.js

```diff
@@ -28,6 +46,7 @@ class App extends Component {
     return(
       <main className='App'>
         <Header />
+        { this.state.error && <h2>{this.state.error}</h2>}
```

Comment:
Great example of conditional rendering.  Maybe the next step could be to add a loading icon in case it's taking a while to load the movie data?